test/bin/scenario.sh

```diff
@@ -1069,6 +1075,169 @@ EOF
     fi
 }
 
+# Implementation of Gingko tests
+run_gingko_tests() {
+    local vmname="${1}"
+    shift
+
+    local full_vmname
+    full_vmname="$(full_vm_name "${vmname}")"
+    if [[ -n "${RUN_HOST_OVERRIDE}" ]]; then
+        vmname="${RUN_HOST_OVERRIDE}"
+        full_vmname="$(full_vm_name "${vmname}")"
+        ip=$(get_vm_ip "${full_vmname}")
+        set_vm_property "${vmname}" "ip" "${ip}"
+        set_vm_property "${vmname}" "ssh_port" "22"
+        set_vm_property "${vmname}" "api_port" "6443"
+        set_vm_property "${vmname}" "lb_port" "5678"
+    fi
+
+    # Save current directory and set up cleanup
+    local original_dir
+    original_dir="$(pwd)"
+    local cleanup_needed=false
+
+    # Check dependencies
+    local missing_deps=()
+    if ! command -v git &> /dev/null; then missing_deps+=("git"); fi
+    if ! command -v go &> /dev/null; then missing_deps+=("go"); fi
+
+    if [[ ${#missing_deps[@]} -gt 0 ]]; then
+        error "Missing required dependencies: ${missing_deps[*]}"
+        for dep in "${missing_deps[@]}"; do
+            record_junit "${vmname}" "${dep}_available" "FAILED"
+        done
+        exit 1
+    fi
+
+    # Check/install oc
+    if ! command -v oc &> /dev/null; then
+        "${ROOTDIR}/scripts/fetch_tools.sh" "oc" || {
+            record_junit "${vmname}" "oc_installed" "FAILED"
+            exit 1
+        }
+    fi
+    record_junit "${vmname}" "dependencies_check" "OK"
+
+    # Set up test environment variables
+    local -r tests_repo_dir="${SCENARIO_INFO_DIR}/${SCENARIO}/openshift-tests-private"
+    local -r tests_repo_url="${OPENSHIFT_TESTS_PRIVATE_REPO:-https://github.com/openshift/openshift-tests-private.git}"
```

Comment:
`openshift-tests-private` repo is a private repository under `openshift` organization, not sure if it's possible to clone it in CI. cc @ggiguash

Reply:
You can ry this:
```suggestion
    local -r tests_repo_url="${OPENSHIFT_TESTS_PRIVATE_REPO:-git@github.com:openshift/openshift-tests-private.git}"
```